examples/python-dataflow/run.rs

```diff
@@ -0,0 +1,41 @@
+use eyre::{bail, Context};
+use std::path::Path;
+
+#[tokio::main]
+async fn main() -> eyre::Result<()> {
+    let root = Path::new(env!("CARGO_MANIFEST_DIR"));
+    std::env::set_current_dir(root.join(file!()).parent().unwrap())
+        .wrap_err("failed to set working dir")?;
+
+    build_package("dora-runtime").await?;
+
+    install_python_dependencies(root).await?;
+
+    dora_coordinator::run(dora_coordinator::Command::Run {
+        dataflow: Path::new("dataflow.yml").to_owned(),
```

Comment:
Maybe we should default to a non-interactive dataflow here? Then we could just run the example on the CI.

Reply:
Yes ok!